runtime/util_core/j9shchelp.c

```diff
@@ -141,80 +165,67 @@ getShcModlevelForJCL(uintptr_t j2seVersion)
 {
 	uint32_t modLevel = 0;
 	switch (j2seVersion) {
-	case J2SE_15 :
-		modLevel = J9SH_MODLEVEL_JAVA5;
-		break;
-	case J2SE_16 :
-		modLevel = J9SH_MODLEVEL_JAVA6;
-		break;
-	case J2SE_17 :
-		modLevel = J9SH_MODLEVEL_JAVA7;
-		break;
 	case J2SE_18 :
 		modLevel = J9SH_MODLEVEL_JAVA8;
 		break;
 	case J2SE_19 :
 		modLevel = J9SH_MODLEVEL_JAVA9;
 		break;
-	case J2SE_V10 :
-		modLevel = J9SH_MODLEVEL_JAVA10;
-		break;
-	case J2SE_V11 :
-		modLevel = J9SH_MODLEVEL_JAVA11;
+	default: 
+		modLevel = JAVA_SPEC_VERSION_FROM_J2SE(j2seVersion);
 		break;
 	}
 	return modLevel;
 }
 
 /**
- * Return the j2se version for a given modification level
- * 
+ * Return the java spec version for a given modification level
  * @param [in] modlevel  The modification level
  * 
  * @return the j2se version or 0 if one cannot be found
  */
 uint32_t
 getJCLForShcModlevel(uintptr_t modlevel)
 {
-	uint32_t j2seVersion = 0;
+	uint32_t javaVersion = 0;
 	switch (modlevel) {
 	case J9SH_MODLEVEL_JAVA5 :
-		j2seVersion = J2SE_15;
+		javaVersion = 5;
 		break;
 	case J9SH_MODLEVEL_JAVA6 :
-		j2seVersion = J2SE_16;
+		javaVersion = 6;
 		break;
 	case J9SH_MODLEVEL_JAVA7 :
-		j2seVersion = J2SE_17;
+		javaVersion = 7;
 		break;
 	case J9SH_MODLEVEL_JAVA8 :
-		j2seVersion = J2SE_18;
+		javaVersion = 8;
 		break;
 	case J9SH_MODLEVEL_JAVA9 :
-		j2seVersion = J2SE_19;
-		break;
-	case J9SH_MODLEVEL_JAVA10 :
-		j2seVersion = J2SE_V10;
+		javaVersion = 9;
 		break;
-	case J9SH_MODLEVEL_JAVA11 :
-		j2seVersion = J2SE_V11;
+	default:
```

Comment:
I'm thinking there should be a case for `6` to handle obsoleted Java 10 cache names that may already exist.